cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -1296,6 +1296,7 @@ def tables_to_csv(
     outgoing_directory,
     tables_to_csv_config_file,
     category_map,
+    filter,
```

Comment:
Please make this parameter optional, so the change doesn't affect existing uses of the stage

Please also document the parameter